tools/cli/adminDBScanCommand.go

```diff
@@ -165,19 +165,18 @@ func AdminDBScan(c *cli.Context) {
 	scaleUpSeconds := c.Int(FlagRPSScaleUpSeconds)
 	scanWorkerCount := c.Int(FlagConcurrency)
 	executionsPageSize := c.Int(FlagPageSize)
-	scanReportRate := c.Int(FlagScanReportRate)
+	scanReportRate := c.Int(FlagReportRate)
 	if numShards < scanWorkerCount {
 		scanWorkerCount = numShards
 	}
 
-	printStopMessage()
```

Comment:
Why remove this?

Reply:
Because it doesn't produce the correct pid to stop the command.

Reply:
Do you want to print a correct stop message then?